src/ucp/wireup/wireup.c

```diff
@@ -1212,7 +1212,36 @@ ucp_wireup_get_dst_rsc_indices(ucp_ep_h ep, ucp_ep_config_key_t *new_key,
     }
 }
 
-static void
+ucs_status_t ucp_wireup_realloc_uct_eps(ucp_ep_h ep, uint32_t new_num_lanes)
+{
+    int num_slow_lanes = (new_num_lanes > UCP_MAX_FAST_PATH_LANES) ?
+                                 new_num_lanes - UCP_MAX_FAST_PATH_LANES :
+                                 0;
```

Comment:
`ucs_max((int)new_num_lanes - (int)UCP_MAX_FAST_PATH_LANES, 0)`
do not init inline because too long

Reply:
seems it's 78 columns